gradle/formatting.gradle

```diff
@@ -19,7 +19,7 @@ allprojects {
             }
         }
         format 'misc', {
-            target '*.md', '**/*.gradle', '**/*.json', '**/*.yaml', '**/*.yml', '**/*.svg', '**/*.properties'
```

Comment:
Do we need this if we dont check in the creds?

Reply:
Sure I can remove it

Reply:
Sure removed it